packages/react/src/component.tsx

```diff
@@ -29,11 +31,23 @@ export function Uploader<TRouter extends FileRouter>(
   );
 }
 
-export function generateComponents<TRouter extends FileRouter>() {
+export function generateComponents<TRouter extends FileRouter>(initOpts: {
+  /**
+   * Absolute URL to the UploadThing API endpoint
+   * @example http://localhost:3000/api/uploadthing
+   */
```

Comment:
silly thing, but also including a "real" url here as a secondary example may be nice, eg: 

``` https://www.example.com/api/uploadthing```

Just because localhost url could be confusing